spec/spec_helper.rb

```diff
@@ -18,8 +18,8 @@
 require 'rspec'
 
 # Require spec support files and shared behavior
-Dir[File.expand_path('../{support,shared}/**/*.rb', __FILE__)].each do |file|
-  require file.chomp('.rb')
+Pathname.glob(Pathname(__dir__).join('{shared,support}', '**', '*.rb')).each do |file|
```

Comment:
I recommend to sort before requirering, I know that in likely these files have no order dependency but its still more deterministic.